docs/plugins/CreatedModifiedDate.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Plugin.CreatedModifiedDate()"
```

Comment:
nit: I think this can just be `CreatedModifiedDate`.

Reply:
Done.